Experimental/MultiplayerUseCases/Assets/Editor/Tutorials/NetworkVariablesVsRPC/50-TestNetworkBehaviour.asset

```diff
@@ -30,49 +30,25 @@ MonoBehaviour:
       m_MaskingSettings:
         m_MaskingEnabled: 1
         m_UnmaskedViews:
-        - m_SelectorType: 0
+        - m_SelectorType: 1
           m_ViewType:
             m_TypeName: UnityEditor.Toolbar, UnityEditor.CoreModule, Version=0.0.0.0,
               Culture=neutral, PublicKeyToken=null
           m_EditorWindowType:
-            m_TypeName: UnityEditor.GameView, UnityEditor.CoreModule, Version=0.0.0.0,
+            m_TypeName: ParrelSync.ClonesManagerWindow, ParrelSync, Version=0.0.0.0,
               Culture=neutral, PublicKeyToken=null
           m_AlternateEditorWindowTypes:
             m_Items: []
           m_MaskType: 0
           m_MaskSizeModifier: 0
-          m_UnmaskedControls:
-          - m_SelectorMode: 5
-            m_SelectorMatchType: 0
-            m_GUIContent:
-              m_Text: 
-              m_Image: {fileID: 0}
-              m_Tooltip: 
-            m_ControlName: ToolbarPlayModePlayButton
-            m_PropertyPath: 
-            m_TargetType:
-              m_TypeName: 
-            m_GUIStyleName: 
-            m_ObjectReference:
-              m_SceneObjectReference:
-                m_SceneGuid: 
-                m_GameObjectGuid: 
-                m_SerializedComponentType:
-                  m_TypeName: 
-                m_ComponentIndex: 0
-                m_AssetObject: {fileID: 0}
-                m_Prefab: {fileID: 0}
-              m_FutureObjectReference: {fileID: 0}
-            m_VisualElementClassName: unity-editor-toolbar__button-strip-element--left
-            m_VisualElementName: Play
-            m_VisualElementTypeName: UnityEditor.Toolbars.EditorToolbarToggle
```

Comment:
I see there were some masking options here that have now been deleted, were these used? If so, they should be kept.

Reply:
It was highlighting the play button and game window before. I thought it might be a bit much highlighting four different areas. But if you want I can add those back into it.

Reply:
yes please, let's add them back since the text of that page explicitly asks you to test things out.